src/openvic-simulation/map/Map.cpp

```diff
@@ -554,6 +568,17 @@ bool Map::load_map_images(fs::path const& province_path, fs::path const& terrain
 	size_t missing = 0;
 	for (size_t idx = 0; idx < province_checklist.size(); ++idx) {
 		Province* province = provinces.get_item_by_index(idx);
+
+		const uint16_t province_index = province->index;
+		if(province_index == Province::NULL_INDEX) {
+			continue;
+		}
+		const uint16_t array_index = province_index-1;
```

Comment:
This whole section isn't needed, `idx` is already the value you need (no subtraction necessary either).

Reply:
You're right. Thanks